src/TuDa.CIMS.Shared/TuDa.CIMS.Shared.csproj

```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <PackageReference Include="ErrorOr" Version="2.0.1" />
     <PackageReference Include="FluentValidation" Version="11.10.0"/>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="9.0.0" />
```

Comment:
🤔 I don't think we should add this here, because the FE does not need to have EF as dependency.